_includes/patterns/race-and-ethnicity-intro.md

```diff
@@ -0,0 +1,8 @@
+{% assign data = site.data.patterns.race %}
+
+<form class="usa-form usa-form--large">
+  <fieldset name="race-and-ethnicity" class="usa-fieldset" aria-multiselectable="true">
+    <legend class="usa-legend">
+        <span class="text-bold">{{ data.legend }}</span> <br/>
```

Comment:
- Are utility classes ok in these pattern previews? I felt this was safer than creating custom styles. 
- Are these text styles appropriate for our pattern examples, or should we stick with default styles? I added some bold and italic text to create some sense of hierarchy, but I am not sure how opinionated we want to be on styling.

Reply:
Resolving this conversation because I removed most of the text-styling utility classes in favor of using `usa-hint`. There are still margin and padding utility classes.